components/shared/Tailwind_Btn/TwButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from 'next/link'
+import { Navigation } from 'components/enum/navigation'
+
+
+interface ButtonProps {
+    href: Navigation;
+    title: string;
+}
+const Button: React.FC<ButtonProps> = ({ href, title }) => {
```

Comment:
At some point we'll want this prop signature to be 
`interface ButtonProps {
    href: Navigation;
    title: string;
} | {
    onClick: () => void;
    title: string;
}`

so that we can pass either a link or an onclick action